plugin-server/src/config/kafka-topics.ts

```diff
@@ -38,6 +38,8 @@ export const KAFKA_CLICKHOUSE_SESSION_REPLAY_EVENTS = `${prefix}clickhouse_sessi
 export const KAFKA_PERFORMANCE_EVENTS = `${prefix}clickhouse_performance_events${suffix}`
 // write heatmap events to ClickHouse
 export const KAFKA_CLICKHOUSE_HEATMAP_EVENTS = `${prefix}clickhouse_heatmap_events${suffix}`
+// write web vitals events to ClickHouse
+export const KAFKA_CLICKHOUSE_WEB_VITALS_EVENTS = `${prefix}clickhouse_web_vitals_events${suffix}`
```

Comment:
TODO reminder to myself, these are autocreated in dev but would need to be terraformed in before merging this